cartography/intel/aws/ec2/instances.py

```diff
@@ -303,6 +323,29 @@ def load_ec2_instance_ebs_volumes(
     )
 
 
+@timeit
+def load_ec2_roles(
+        neo4j_session: neo4j.Session,
+        role_data: List[Dict[str, Any]],
+        region: str,
+        current_aws_account_id: str,
+        update_tag: int,
+) -> None:
+    ingest_role_instance_relations = """
+    UNWIND $roles as role
+    MERGE (instance:EC2Instance {id: role.InstanceId}), (roleNode:AWSRole {arn: role.IamRoles})
```

Comment:
role object doesn’t have `InstanceId` key
Can you check API document?
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/iam/client/get_instance_profile.html